Brink Innovation/Example CHT application/disease-surveillance/nools-extras.js

```diff
@@ -0,0 +1,63 @@
+function getField(report, fieldPath) {
```

Comment:
This function is already available in [cht-nootils](https://github.com/medic/cht-nootils/blob/main/src/nootils.js#L102).